docs/gridstatus_download.md

```diff
@@ -0,0 +1,54 @@
+# Grid Status Data Download
+
+This script downloads LMP (Locational Marginal Pricing) data from [Grid Status](https://www.gridstatus.io/) for use in Hercules simulations. Grid Status provides comprehensive electricity market data including real-time and historical pricing information.
+
+## What is Grid Status?
+
+[Grid Status](https://www.gridstatus.io/) is a platform that provides electricity market data including:
+- Real-time and historical LMP data
+- Market operations data
+- Grid status information
+- Power system reliability data
+
+## API Key Setup
+
+To use this script, you'll need an API key from Grid Status:
+
+1. Visit [Grid Status API Settings](https://www.gridstatus.io/settings/api)
+2. Sign up for an account if you don't have one
+3. Generate an API key
+4. Set the API key as an environment variable or configure it in your Grid Status client
```

Comment:
Mentioning the specific steps for setting as an environment variable would be helpful. The README file here clarified it for me: https://github.com/gridstatus/gridstatusio